modules/virt-golden-images.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-golden-images_{context}"]
+= Using golden images as a boot source
+
+// Lead paragraph here
+
+== What are golden images?
+
+A golden image is a pre-configured snapshot of a virtual machine (VM) that you can use as a resource to deploy new VMs. For example, you can use golden images to provision the same system environment consistently and deploy systems more quickly and efficiently.
+
+== How do golden images work?
+
+Golden images are created by installing and configuring an operating system and software applications on a reference machine or virtual machine. This includes setting up the system, installing required drivers, applying patches and updates, and configuring specific options and preferences.
+
+After the golden image is created, it is saved as a template or image file that can be replicated and deployed across multiple clusters. The golden image can be updated by its maintainer periodically to incorporate necessary software updates and patches, ensuring that the image remains up to date and secure, and newly created VMs are based on this updated image.
+
+== Red Hat implementation of golden images
+
+You can create a VM on RHEL by using golden images available in the Red Hat Container Registry.
+
+. Red Hat publishes RHEL golden images as container disks in the Red Hat Container Registry. These images can be consumed from connected clusters that have OpenShift Virtualization installed.
+. When a cluster administrator provisions an image from the registry, the container disk is imported into a PVC on the cluster. This provides stability and enables fast VM creation.

```

Comment:
I am pretty sure a cluster admin does not provision an image from a registry. Red Hat creates the image and pushes it to the registry. The CDI pulls the image from the registry by using the DataImportCron.
I'm not sure why this step would provide stability and fast VM creation.